feat(Translations): Translations for YouTube and Twitch

LisoUseInAIKyrios opened the pull request:
Removed usage of `StringResource`, and moved all strings into Strings.xml files which allows easy localization.

Added translations for a few random languages (machine translated to show it works.  Preferably should be replaced with Crowdin or some other human translated source)

Translation files are bundled automatically, just add or remove the translated Strings.xml files to either `youtube/translation` or `twitch/translation`.

Hebrew:
![hebrew settings](https://github.com/revanced/revanced-patches/assets/118716522/8422b3c2-c144-49cc-b189-eb6fcfd23275)
![hebrew sponsorblock](https://github.com/revanced/revanced-patches/assets/118716522/1f312d94-39cc-4d0c-8564-f411caf89cd5)


French (_ooh là là_):
![french sponsorblock](https://github.com/revanced/revanced-patches/assets/118716522/975224ac-8079-4a82-affa-01cdf9bc8280)


French Twitch:
![french twitch](https://github.com/revanced/revanced-patches/assets/118716522/29eaf898-679e-4282-aab5-c77956991621)


[integration changes](https://github.com/revanced/revanced-integrations/pull/420)

Commit:
move youtube resources into subdirectory

Commit:
removed all StringResource usage for YouTube

Commit:
sorting by title now done in integrations

Commit:
youtube translations

Commit:
no longer needed

Commit:
use same minimum version as YouTube.  Not that it matters.

Commit:
cleanup

Commit:
adding annotation, moving code

Commit:
extract strings for twitch

Commit:
rename class

Commit:
hide the last bits of StringResource

Commit:
twitch translations stubs

Commit:
no functional changes: renamed classes

Commit:
fixing escaping

Commit:
fixing twitch

0xrxL commented:
Crowdin is a better solution imho.

Better to translator too (with its UX).

0xrxL commented:
@oSumAtrIX Do you think Crowdin it's a good solution ReVanced translations?

Commit:
cleanup

Commit:
simplify array resources

Commit:
fixing stuff

Commit:
comments are in integrations

Commit:
more clear

Commit:
strings that don't need, or should not be translated

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/ad/getpremium/bytecode/patch/HideGetPremiumVideoAdvertisementPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/autorepeat/patch/AutoRepeatPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/hdrbrightness/patch/HDRBrightnessPatch.kt

Commit:
fixing merge

Commit:
moving YouTube microg strings to localized file

Commit:
fixing merge

Commit:
more non translatable text

Commit:
added option to throw an exception if destination files exist.

Commit:
Merge branch 'dev' into localization

Commit:
move non-translatable strings into separate file.

Commit:
comments

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

Commit:
SponsorBlock has removed the leaderboard (the api end point no longer exists), and the website leaderboard has been broken for what seems like forever.

Commit:
no functional changes: renamed classes

Commit:
no functional changes: renamed classes

Commit:
tiktok translations

Commit:
Revert "tiktok translations"

This reverts commit 853e882a1528e4c191f293f4b27739220ba39661.

Commit:
twitch machine translations

Commit:
turned off, for now

Commit:
Merge branch 'dev' into localization

Commit:
Merge branch 'dev' into localization

Commit:
Verify the regex matches the end of the path, in case the user likes to use crazy names for subdirectories

Commit:
Merge branch 'dev' into localization

Commit:
Merge branch 'dev' into localization

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/ad/general/resource/patch/HideAdsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/artistcards/patch/HideArtistCardsPatch.kt

Commit:
fixing merge

Commit:
no point translating this dead patch.

Commit:
use same ordering as shown in the app.

Commit:
use revanced prefix

Commit:
use revanced prefix

Commit:
fixing stuff

Commit:
comments

Commit:
making resource names more consistent

Commit:
Merge branch 'dev' into localization

Commit:
logging out is no longer needed nor works, as YouTube now always gives the new UI.

Commit:
Merge remote-tracking branch 'origin/localization' into localization

Commit:
adjust strings

Commit:
Merge branch 'dev' into localization

Commit:
fixing RTL languages

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/comments/patch/CommentsPatch.kt

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/patch/HideLayoutComponentsPatch.kt

Commit:
finishing merge

Commit:
committing missing resources lost in refactoring

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

Commit:
Merge branch 'dev' into localization

Commit:
Adding last untranslated strings.
Verified that all YouTube strings are now extracted to Strings.xml.

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/patch/SpoofSignatureVerificationPatch.kt

Commit:
finishing merge

Commit:
downloading appears to be working again.

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/misc/debugging/patch/DebuggingPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/bytecode/fingerprints/SetThemeFingerprint.kt

Commit:
this should be the last non-localized string

Commit:
Merge branch 'revanced:main' into localization

Commit:
Merge branch 'dev' into localization

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/patch/SpoofSignatureVerificationPatch.kt

Commit:
finishing merge

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/resources/sponsorblock/host/values/strings.xml

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/patch/HideLayoutComponentsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/videobuffer/annotations/CustomVideoBufferCompatibility.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/oldqualitylayout/patch/OldQualityLayoutResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/speed/custom/patch/CustomVideoSpeedPatch.kt

Commit:
Merge branch 'dev' into localization

0xrxL commented:
@oSumAtrIX What's still missing on this PR before it can be merged?

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/tiktok/feedfilter/patch/FeedFilterPatch.kt
#	src/main/kotlin/app/revanced/patches/tiktok/interaction/downloads/patch/DownloadsPatch.kt
#	src/main/kotlin/app/revanced/patches/tiktok/misc/integrations/patch/TikTokIntegrationsPatch.kt
#	src/main/kotlin/app/revanced/patches/tiktok/misc/settings/patch/TikTokSettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/tiktok/misc/spoof/sim/patch/SpoofSimPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/ad/audio/patch/AudioAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/ad/embedded/patch/EmbeddedAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/ad/video/patch/VideoAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/chat/antidelete/patch/ShowDeletedMessagesPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/chat/autoclaim/patch/AutoClaimChannelPointsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/debug/patch/DebugModePatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/misc/settings/bytecode/patch/TwitchSettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/misc/settings/resource/patch/TwitchSettingsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/ad/getpremium/bytecode/patch/HideGetPremiumVideoAdvertisementPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/ad/video/patch/VideoAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/copyvideourl/resource/patch/CopyVideoUrlResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/downloads/resource/patch/ExternalDownloadsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/swipecontrols/patch/resource/SwipeControlsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/autocaptions/patch/AutoCaptionsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/autoplay/patch/HideAutoplayButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/captions/patch/HideCaptionsButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/cast/patch/HideCastButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/navigation/patch/NavigationButtonsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/player/hide/patch/HidePlayerButtonsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/albumcards/resource/patch/AlbumCardsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/breakingnews/resource/patch/BreakingNewsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/crowdfundingbox/resource/patch/CrowdfundingBoxResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/endscreencards/resource/patch/HideEndscreenCardsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/loadmorebutton/resource/patch/HideLoadMoreButtonResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/personalinformation/resource/patch/HideEmailAddressResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/player/overlay/resource/patch/HidePlayerOverlayResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/shorts/resource/patch/HideShortsComponentsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/time/patch/HideTimestampPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/watchinvr/patch/WatchInVRPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/watermark/patch/HideWatermarkPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/panels/fullscreen/remove/patch/FullscreenPanelsRemoverPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/panels/popup/patch/PlayerPopupPanelsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/returnyoutubedislike/resource/patch/ReturnYouTubeDislikeResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/searchbar/patch/WideSearchbarPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/resource/patch/SponsorBlockResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/bytecode/patch/SpoofAppVersionPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/startupshortsreset/patch/DisableShortsOnStartupPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/tabletminiplayer/patch/TabletMiniPlayerPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/debugging/patch/DebuggingPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/microg/patch/resource/MicroGResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/bytecode/patch/YouTubeSettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/resource/patch/YouTubeSettingsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/quality/patch/RememberVideoQualityPatch.kt

Commit:
resolving merge specific conflicts

Commit:
fixing merge

oSumAtrIX reviewed (changes requested):
The only problem with this PR is that it breaks the separation of patches. Now a single file manages multiple patches. This means the apps will contain traces of patches that should not be there, to begin with. A similar issue is seen with integrations currently, which will eventually be solved once integrations are coupled with their respective patches. Is it possible to separate the files so that traces of patches I did not apply do not appear in the patched app? I was following the `host` convention so far for that reason. Some patches delivered resources via a "host". The respective patch then uses the host to add the patch's respective resources to the app. This may be a huge task to refactor for this PR which is why if it is, do not hesitate to suggest a solution that can be approached together.

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/video/speed/custom/patch/CustomPlaybackSpeedPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/speed/remember/patch/RememberPlaybackSpeedPatch.kt

Commit:
fixing merge

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/tiktok/misc/settings/patch/TikTokSettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/theme/resource/ThemeResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/patch/SpoofSignatureVerificationPatch.kt

Commit:
merge specific updates changes

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/bytecode/patch/SpoofAppVersionPatch.kt

Commit:
merge specific changes

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

Commit:
Use latest patch naming

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/bytecode/patch/SpoofAppVersionPatch.kt

Commit:
merge specific changes

Commit:
fixing merge

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/tiktok/misc/settings/patch/TikTokSettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/misc/settings/bytecode/patch/TwitchSettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/seekbar/patch/EnableSeekbarTappingResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/swipecontrols/patch/resource/SwipeControlsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/comments/patch/CommentsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/filterbar/patch/HideFilterBarResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/infocards/resource/patch/HideInfocardsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/shorts/resource/patch/HideShortsComponentsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/bytecode/patch/YouTubeSettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/hdrbrightness/patch/HDRBrightnessPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/speed/custom/patch/CustomPlaybackSpeedPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/speed/remember/patch/RememberPlaybackSpeedPatch.kt

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/watchinvr/patch/WatchInVRPatch.kt

Commit:
merge specific changes

Commit:
Merge remote-tracking branch 'upstream/dev' into localization

Commit:
remove deprecated version annotation

LisoUseInAIKyrios committed:
fixing string

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/patch/SpoofSignatureVerificationResourcePatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/action/patch/HideButtonsPatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/shared/settings/preference/impl/NonInteractivePreference.kt
#	src/main/kotlin/app/revanced/patches/shared/settings/preference/impl/StringResource.kt
#	src/main/kotlin/app/revanced/patches/shared/settings/preference/impl/SwitchPreference.kt
#	src/main/kotlin/app/revanced/patches/tiktok/feedfilter/patch/FeedFilterPatch.kt
#	src/main/kotlin/app/revanced/patches/tiktok/interaction/downloads/patch/DownloadsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/ad/embedded/patch/EmbeddedAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/chat/antidelete/patch/ShowDeletedMessagesPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/chat/autoclaim/patch/AutoClaimChannelPointsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/misc/settings/bytecode/patch/TwitchSettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/ad/getpremium/bytecode/patch/HideGetPremiumVideoAdvertisementPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/ad/video/patch/VideoAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/copyvideourl/resource/patch/CopyVideoUrlResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/downloads/resource/patch/ExternalDownloadsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/seekbar/patch/EnableSeekbarTappingResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/swipecontrols/patch/resource/SwipeControlsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/autocaptions/patch/AutoCaptionsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/action/patch/HideButtonsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/autoplay/patch/HideAutoplayButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/captions/patch/HideCaptionsButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/cast/patch/HideCastButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/navigation/patch/NavigationButtonsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/player/hide/patch/HidePlayerButtonsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/albumcards/resource/patch/AlbumCardsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/breakingnews/resource/patch/BreakingNewsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/crowdfundingbox/resource/patch/CrowdfundingBoxResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/endscreencards/resource/patch/HideEndscreenCardsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/filterbar/patch/HideFilterBarResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/floatingmicrophone/patch/HideFloatingMicrophoneButtonResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/infocards/resource/patch/HideInfocardsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/loadmorebutton/bytecode/patch/HideLoadMoreButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/loadmorebutton/resource/patch/HideLoadMoreButtonResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/personalinformation/resource/patch/HideEmailAddressResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/player/flyoutmenupanel/patch/HidePlayerFlyoutMenuPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/player/overlay/resource/patch/HidePlayerOverlayResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/shorts/resource/patch/HideShortsComponentsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/time/patch/HideTimestampPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/watermark/patch/HideWatermarkPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/panels/fullscreen/remove/patch/FullscreenPanelsRemoverPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/panels/popup/patch/PlayerPopupPanelsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/returnyoutubedislike/resource/patch/ReturnYouTubeDislikeResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/searchbar/patch/WideSearchbarPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/resource/patch/SponsorBlockResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/bytecode/patch/SpoofAppVersionPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/startupshortsreset/patch/DisableShortsOnStartupPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/tabletminiplayer/patch/TabletMiniPlayerPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/autorepeat/patch/AutoRepeatPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/debugging/patch/DebuggingPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/patch/SpoofSignatureVerificationPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/patch/SpoofSignatureVerificationResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/links/open/patch/OpenLinksExternallyPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/microg/patch/resource/MicroGResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/minimizedplayback/patch/MinimizedPlaybackPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/bytecode/fingerprints/SetThemeFingerprint.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/bytecode/patch/YouTubeSettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/resource/patch/YouTubeSettingsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/zoomhaptics/patch/ZoomHapticsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/hdrbrightness/patch/HDRBrightnessPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/quality/patch/RememberVideoQualityPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/speed/custom/patch/CustomPlaybackSpeedPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/speed/remember/patch/RememberPlaybackSpeedPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/videoqualitymenu/patch/OldVideoQualityMenuResourcePatch.kt
#	src/main/kotlin/app/revanced/util/resources/ResourceUtils.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/tiktok/misc/integrations/annotations/TikTokIntegrationsCompatibility.kt
#	src/main/kotlin/app/revanced/patches/tiktok/misc/integrations/patch/TikTokIntegrationsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/integrations/patch/IntegrationsPatch.kt

LisoUseInAIKyrios committed:
distinct names from other integration patches

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/twitch/misc/settings/bytecode/patch/TwitchSettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/patch/HideLayoutComponentsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/player/overlay/resource/patch/CustomPlayerOverlayOpacityResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/links/open/patch/OpenLinksExternallyPatch.kt
#	src/main/kotlin/app/revanced/util/resources/ResourceUtils.kt

LisoUseInAIKyrios committed:
fixing merge conflicts

LisoUseInAIKyrios committed:
fixing merge

LisoUseInAIKyrios committed:
revert "AppNameSettingsPatch" back to just "SettingsPatch"

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/patch/HideLayoutComponentsPatch.kt

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/patch/HideLayoutComponentsPatch.kt

LisoUseInAIKyrios committed:
extracted strings to individual patch files

LisoUseInAIKyrios committed:
Code to merge English strings into a single xml file, and gradle task to run it all.

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/patch/HideLayoutComponentsPatch.kt

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/twitch/ad/embedded/patch/EmbeddedAdsPatch.kt

LisoUseInAIKyrios committed:
Moved Twitch strings to individual files

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
extract string

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/shared/settings/preference/impl/SwitchPreference.kt
#	src/main/kotlin/app/revanced/patches/tiktok/misc/integrations/patch/IntegrationsPatch.kt
#	src/main/kotlin/app/revanced/patches/tiktok/misc/settings/SettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/ad/audio/AudioAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/ad/embedded/EmbeddedAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/ad/video/VideoAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/chat/antidelete/ShowDeletedMessagesPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/chat/autoclaim/AutoClaimChannelPointsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/debug/DebugModePatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/misc/integrations/patch/IntegrationsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/misc/settings/SettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/twitch/misc/settings/resource/patch/SettingsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/ad/general/HideAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/ad/general/HideAdsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/ad/getpremium/HideGetPremiumVideoAdvertisementPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/ad/video/VideoAdsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/copyvideourl/CopyVideoUrlResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/downloads/ExternalDownloadsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/seekbar/EnableSeekbarTappingPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/seekbar/EnableSeekbarTappingResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/swipecontrols/SwipeControlsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/autocaptions/AutoCaptionsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/action/HideButtonsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/autoplay/HideAutoplayButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/captions/HideCaptionsButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/cast/HideCastButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/navigation/NavigationButtonsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/buttons/player/hide/HidePlayerButtonsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/albumcards/AlbumCardsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/breakingnews/BreakingNewsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/comments/CommentsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/crowdfundingbox/CrowdfundingBoxResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/endscreencards/HideEndscreenCardsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/filterbar/HideFilterBarResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/floatingmicrophone/HideFloatingMicrophoneButtonResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/HideLayoutComponentsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/infocards/HideInfocardsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/loadmorebutton/HideLoadMoreButtonResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/loadmorebutton/bytecode/patch/HideLoadMoreButtonPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/personalinformation/HideEmailAddressResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/player/flyoutmenupanel/HidePlayerFlyoutMenuPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/seekbar/HideSeekbarPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/shorts/HideShortsComponentsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/time/HideTimestampPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/watermark/HideWatermarkPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/panels/popup/PlayerPopupPanelsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/player/overlay/CustomPlayerOverlayOpacityResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/returnyoutubedislike/ReturnYouTubeDislikeResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/searchbar/WideSearchbarPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/seekbar/SeekbarPreferencesPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/SponsorBlockResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/SpoofAppVersionPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/startupshortsreset/DisableShortsOnStartupPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/tablet/EnableTabletLayoutPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/tabletminiplayer/TabletMiniPlayerPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/theme/ThemeResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/thumbnails/AlternativeThumbnailsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/autorepeat/AutoRepeatPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/debugging/DebuggingPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/SpoofSignatureVerificationPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/SpoofSignatureVerificationResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/integrations/IntegrationsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/links/OpenLinksExternallyPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/minimizedplayback/MinimizedPlaybackPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/SettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/SettingsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/fingerprints/SetThemeFingerprint.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/zoomhaptics/ZoomHapticsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/hdrbrightness/HDRBrightnessPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/quality/RememberVideoQualityPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/speed/custom/CustomPlaybackSpeedPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/speed/remember/RememberPlaybackSpeedPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/videoqualitymenu/OldVideoQualityMenuResourcePatch.kt
#	src/main/kotlin/app/revanced/util/resources/ResourceUtils.kt

LisoUseInAIKyrios committed:
fixing merge

semantic-release-bot committed:
chore(release): 2.191.0-dev.6 [skip ci]

# [2.191.0-dev.6](https://github.com/ReVanced/revanced-patches/compare/v2.191.0-dev.5...v2.191.0-dev.6) (2023-09-22)

### Features

* **Tumblr:** Add `Disable Tumblr Live` patch ([#2987](https://github.com/ReVanced/revanced-patches/issues/2987)) ([bf1f9dc](https://github.com/ReVanced/revanced-patches/commit/bf1f9dc799705679d17973e689165ab1bff327cd))
* **YouTube - Spoof app version:** add version 18.20.39 ([#3001](https://github.com/ReVanced/revanced-patches/issues/3001)) ([f14c5e7](https://github.com/ReVanced/revanced-patches/commit/f14c5e79792f62fb060dd0eebbd3dd7157a08a98))

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
cleanup of merge

LisoUseInAIKyrios committed:
fixing initialization error

LisoUseInAIKyrios committed:
Fixing missing string.  Desperately need to hook aapt up to the console logging because these errors currently give no feedback and are a huge waste of time to figure out.

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	CHANGELOG.md

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/SpoofSignatureVerificationResourcePatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/SpoofSignaturePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/SpoofSignatureResourcePatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/SpoofSignaturePatch.kt

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/loadmorebutton/HideLoadMoreButtonResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/SpoofSignaturePatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
refactor: rename string file to match pat

LisoUseInAIKyrios committed:
merge specific updates

LisoUseInAIKyrios committed:
refactor: no functional changes: rearrange strings to the same order as used.

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/resources/sponsorblock/host/values/strings.xml

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/shared/settings/util/AbstractPreferenceScreen.kt
#	src/main/kotlin/app/revanced/patches/youtube/ad/getpremium/HideGetPremiumPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/seekbar/EnableSeekbarTappingResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/HideLayoutComponentsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/loadmorebutton/HideLoadMoreButtonResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/watermark/HideWatermarkPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/SpoofAppVersionPatch.kt
#	src/main/resources/youtube/settings/host/values/ReturnYouTubeDislike.xml

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Validate strings have been merged before using, since resource compilation errors currently don't give any feedback.

LisoUseInAIKyrios committed:
fix importing wrong class

LisoUseInAIKyrios committed:
Merge branch 'dev' into localization

LisoUseInAIKyrios committed:
Update src/main/kotlin/app/revanced/util/resources/ResourceUtils.kt

Co-authored-by: oSumAtrIX <johan.melkonyan1@web.de>

LisoUseInAIKyrios committed:
importing duplicate strings almost always indicates an problem

LisoUseInAIKyrios committed:
description

LisoUseInAIKyrios committed:
Comments

LisoUseInAIKyrios committed:
Sorting is now done in integrations based on the localized language in use

LisoUseInAIKyrios committed:
correctly handle `formatted="true"`

oSumAtrIX committed:
refactor: Slight refactoring

LisoUseInAIKyrios committed:
no functional changes: rename methods and comments

LisoUseInAIKyrios committed:
removing as it's generated.

LisoUseInAIKyrios committed:
remove gradle merge task, as it appears CrowdIn will work as desired without this.

LisoUseInAIKyrios committed:
No functional changes: renamed Utils classes.

LisoUseInAIKyrios committed:
fixing merge

LisoUseInAIKyrios committed:
fixing overlooked refactor

LisoUseInAIKyrios committed:
Comments

oSumAtrIX committed:
refactor: Use `SettingsPatch` instead of `SettingsResourcePath`

Because the patches depend on `SettingsPatch`.

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/interaction/seekbar/DisablePreciseSeekingGesturePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/branding/CustomBrandingPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/theme/ThemeBytecodePatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
extract Strings bundling into it's own patch

LisoUseInAIKyrios committed:
fix typo

LisoUseInAIKyrios committed:
comments

LisoUseInAIKyrios committed:
removed resource callback that is no longer needed

LisoUseInAIKyrios committed:
comments

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/player/flyoutmenupanel/HidePlayerFlyoutMenuPatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
No longer need to check since resource compilation now shows errors

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/seekbar/RestoreOldSeekbarThumbnailsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/videoqualitymenu/RestoreOldVideoQualityMenuResourcePatch.kt
#	src/main/resources/sponsorblock/host/values/strings.xml

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/SpoofSignaturePatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/HideLayoutComponentsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/links/BypassURLRedirectsPatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/startupshortsreset/DisableResumingShortsOnStartupPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/microg/MicroGResourcePatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
feat(YouTube):Add `Disable rolling number animations` patch (#3298)

oSumAtrIX committed:
fix(Twitch - Settings): Constrain to last working version

oSumAtrIX committed:
fix(Yuka - Unlock premium): Constrain to last working version

semantic-release-bot committed:
chore(release): 2.198.1-dev.2 [skip ci]

## [2.198.1-dev.2](https://github.com/ReVanced/revanced-patches/compare/v2.198.1-dev.1...v2.198.1-dev.2) (2023-11-19)

### Bug Fixes

* **Twitch - Settings:** Constrain to last working version ([a3e275b](https://github.com/ReVanced/revanced-patches/commit/a3e275b79adcba1ed85f359e0905759e466e8b0c))
* **Yuka - Unlock premium:** Constrain to last working version ([9b855f0](https://github.com/ReVanced/revanced-patches/commit/9b855f0b78d6c10190a433901841a7c0d9391f93))

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/shared/settings/util/AbstractPreferenceScreen.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/HideLayoutComponentsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/seekbar/HideSeekbarPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/seekbar/SeekbarPreferencesPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/announcements/AnnouncementsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/SpoofSignaturePatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	CHANGELOG.md
#	gradle.properties
#	patches.json
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/rollingnumber/DisableRollingNumberAnimationPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/seekbar/HideSeekbarPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/announcements/AnnouncementsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/privacy/RemoveTrackingQueryParameterPatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/youtube/layout/hide/general/HideLayoutComponentsPatch.kt

LisoUseInAIKyrios committed:
merge specific changes

LisoUseInAIKyrios committed:
extract strings

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/twitch/misc/settings/SettingsPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/seekbar/RestoreOldSeekbarThumbnailsPatch.kt

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

LisoUseInAIKyrios committed:
Merge remote-tracking branch 'upstream/dev' into localization

# Conflicts:
#	src/main/kotlin/app/revanced/patches/shared/settings/AbstractSettingsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/copyvideourl/CopyVideoUrlResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/downloads/ExternalDownloadsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/interaction/swipecontrols/SwipeControlsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/returnyoutubedislike/ReturnYouTubeDislikeResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/SponsorBlockResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/microg/MicroGResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/minimizedplayback/MinimizedPlaybackPatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/misc/settings/SettingsResourcePatch.kt
#	src/main/kotlin/app/revanced/patches/youtube/video/speed/remember/RememberPlaybackSpeedPatch.kt
#	src/main/kotlin/app/revanced/util/microg/MicroGResourceHelper.kt
#	src/main/kotlin/app/revanced/util/resources/ResourceUtils.kt
#	src/main/resources/youtube/stringspatch/host/values/ReturnYouTubeDislike.xml

LisoUseInAIKyrios committed:
added shared string patch support.  added better exception logging if resources do not exist.

LisoUseInAIKyrios committed:
rearrange packages

oSumAtrIX commented:
Hey, just letting you know that we have renamed repositories. Please close this PR and re-open it [here](https://github.com/ReVanced/revanced-patches/pulls). PS: No additional changes are needed to your remotes.

LisoUseInAIKyrios commented:
Pull request moved to https://github.com/ReVanced/revanced-patches/pull/2440
